app/models/twitter_user.rb

```diff
@@ -120,32 +120,32 @@ def build_relations
       ]
       if without_friends
         candidates = candidates.slice(2, candidates.size - 2)
-        _statuses, _search_results, _, _mentions, _favorites = client.fetch_parallelly(candidates)
+        _statuses, _search_results, _, _mentions, _favorites = client._fetch_parallelly(candidates)
         _friends = _followers = []
       else
-        _friends, _followers, _statuses, _search_results, _, _mentions, _favorites = client.fetch_parallelly(candidates)
+        _friends, _followers, _statuses, _search_results, _, _mentions, _favorites = client._fetch_parallelly(candidates)
       end
     else
       candidates = [
-        {method: :friends_advanced, args: [uid_i]},
-        {method: :followers_advanced, args: [uid_i]},
+        {method: :friends_parallelly, args: [uid_i]},
+        {method: :followers_parallelly, args: [uid_i]},
         {method: :user_timeline, args: [uid_i]},
         {method: :search, args: [search_query]},
         {method: :favorites, args: [uid_i]}
       ]
       if without_friends
         candidates = candidates.slice(2, candidates.size - 2)
-        _statuses, _search_results, _favorites = client.fetch_parallelly(candidates)
+        _statuses, _search_results, _favorites = client._fetch_parallelly(candidates)
         _friends = _followers = []
       else
-        _friends, _followers, _statuses, _search_results, _favorites = client.fetch_parallelly(candidates)
+        _friends, _followers, _statuses, _search_results, _favorites = client._fetch_parallelly(candidates)
```

Comment:
__[rubocop]__
* Line is too long. [107/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)